src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -444,14 +448,14 @@ public void notifications_clickOnNotificationsLogo() {
     waitForElementClickableByElement(notificationsShowNotificationsLogo);
     notificationsShowNotificationsLogo.click();
     PageObjectLogging.log("notifications_clickOnNotificationsLogo",
-        "click on notifications logo on the upper right corner", true, driver);
+                          "click on notifications logo on the upper right corner", true, driver);
   }
 
   public void notifications_showNotifications() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00100)